src/main/java/core/basesyntax/figures/IPrintable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.figures;
+
+public interface IPrintable {

```

Comment:
Debatable topic but I would not recommend you put the prefix "I" before interface names, this is not common practice at least for Java. Here's an interesting discussion on it if you want to dig deeper into the pros and cons of this choice https://stackoverflow.com/questions/541912/interface-naming-in-java